frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.ts

```diff
@@ -214,9 +215,14 @@ export class TeamPlannerComponent extends UntilDestroyedMixin implements OnInit,
               month: 'long',
               day: 'numeric',
             },
-            initialView: 'resourceTimelineWeekDaysOnly',
+            initialView: (this.calendar.initialView || 'resourceTimelineWeek') as string,
```

Comment:
Feels like it's time to refactor this whole default calendar options out into their own file so the component becomes more readable.

Reply:
Yes works for me. We'd still have to pass in the calendar service as a dependency, as I haven't found a way to merge the default view from cview vs. the fallback otherwise.

Reply:
Trying it out, I don't think we gain much from this as you have bound methods for the events, and then would split up the configuration even more.